css/style.css

```
@@ -0,0 +1,818 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  position: relative;
+  font-family: Poppins, sans-serif;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 30px;
+  background: url(../img/decore.svg) no-repeat top right;
+  background-position-y: -140px;
+  background-size: 50%;
+}
+
+h1 {
+  color: #181E4B;
+  font-family: Volkhov;
+  font-size: 84px;
+  font-weight: 700;
+  line-height: 89px;
+  letter-spacing: -3.36px;
+}
+
+h2 {
+  color: #14183E;
+  font-family: Volkhov;
+  font-size: 50px;
+  font-weight: 700;
+  text-transform: capitalize;
+}
+
+.body-decor {
+  position: absolute;
+  top: 100px;
+  left: -200px;
+  width: 400px;
+  height: 400px;
+  display: block;
+  border-radius: 50%;
+  opacity: 0.3;
+  background: #D5AEE4;
+  filter: blur(75px);
+}
+
+.container {
+  max-width: 1440px;
+  margin: 0 auto;
+}
+
+.header-wrap {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 48px 140px 23px;
+}
+
+.header-nav {
+  margin-left: auto;
+  margin-right: 85px;
+}
+
+.header-list {
+  display: flex;
+  align-items: center;
+  gap: 60px;
+  list-style: none;
+}
+
+.header-menu-link {
+  text-decoration: none;
+  color: #212832;
+  font-size: 17px;
+}
+
+.header-btn {
+  color: #212832;
+  font-family: Poppins;
+  font-size: 17px;
+  font-weight: 500;
+  background: none;
+  border: 0;
+}
+
+.border {
+  padding: 9px 21px;
+  border: 1px solid #212832;
+  border-radius: 5px;
+  margin-left: 52px;
+}
+
+.header-lang {
+  position: relative;
+  margin-left: 42px;
+}
+
+.header-lang::after {
+  position: absolute;
+  content: '>';
+  top: 0;
+  right: -15px;
+  transform: rotateZ(90deg);
+}
+
+.hero {
+  position: relative;
+  padding: 0 100px 0 150px;
+  margin-bottom: 97px;
+}
+
+.hero-content {
+  position: absolute;
+  left: 150px;
+  top: 110px;
+}
+
+.hero-subtitle {
+  color: #DF6951;
+  font-family: Poppins;
+  font-size: 20px;
+  font-weight: 700;
+  text-transform: uppercase;
+  margin: 0;
+}
+
+.hero-title {
+  position: relative;
+  max-width: 545px;
+  margin: 24px 0 30px;
+}
+
+.hero-title::after {
+  position: absolute;
+  width: 330px;
+  height: 15px;
+  content: '';
+  left: 245px;
+  top: 70px;
+  background: url(../img/Decore2.svg) no-repeat;
+  z-index: -1;
+}
```

обычно всегда все пишут content на первом месте, так как если его провтыкать - то псевдоэлемент вообще работать не будет ( вынес только content на первое место )

```suggestion
.hero-title::after {
  content: '';
  position: absolute;
  width: 330px;
  height: 15px;
  left: 245px;
  top: 70px;
  background: url(../img/Decore2.svg) no-repeat;
  z-index: -1;
}
```